### ISSUE TITLE
qemu-x86_64-static: Could not open ‘/lib64/ld-linux-x86-64.so.2’: No such file or directory

### ISSUE THREAD

**Issue by Marietto2008** [antmicro/kvm-aosp-jetson-nano#12]
I’m following your tutorial and I haven’t been able to finish it because of an error that seems easy to fix. Any idea what's wrong ?

Checking out projects: 100% (747/747), done.
repo sync has finished successfully.
root@ziomario-desktop:# ls
Android.bp cts frameworks packages test
art dalvik hardware pdk toolchain
bionic developers kernel platform_testing tools
bootable development libcore prebuilts
bootstrap.bash device libnativehelper sdk
build external Makefile system
root@ziomario-desktop:# source build/envsetup.sh
root@ziomario-desktop:# lunch aosp_arm64-eng

qemu-x86_64-static: Could not open ‘/lib64/ld-linux-x86-64.so.2’: No such file or directory

**Comment by pgielda** [comment]
Can you change your Linux language settings to English, re-run and paste results? Unfortunately half of the log is in Italian...

**Comment by Marietto2008** [comment]
repo sync has finished successfully.
root@ziomario-Z390-AORUS-PRO:/home/ziomario/Scrivania/Data/Nano/I9/aosp# source build/envsetup.sh
root@ziomario-Z390-AORUS-PRO:/home/ziomario/Scrivania/Data/Nano/I9/aosp# lunch aosp_arm64-eng
[W][2021-03-07T17:05:09+0100][37844] void cmdline::logParams(nsjconf_t *)():250 Process will be UID/EUID=0 in the global user namespace, and will have user root-level access to files
[W][2021-03-07T17:05:09+0100][37844] void cmdline::logParams(nsjconf_t *)():260 Process will be GID/EGID=0 in the global user namespace, and will have group root-level access to files

============================================
PLATFORM_VERSION_CODENAME=REL
PLATFORM_VERSION=10
TARGET_PRODUCT=aosp_arm64
TARGET_BUILD_VARIANT=eng
TARGET_BUILD_TYPE=release
TARGET_ARCH=arm64
TARGET_ARCH_VARIANT=armv8-a
TARGET_CPU_VARIANT=generic
TARGET_2ND_ARCH=arm
TARGET_2ND_ARCH_VARIANT=armv8-a
TARGET_2ND_CPU_VARIANT=generic
HOST_ARCH=x86_64
HOST_2ND_ARCH=x86
HOST_OS=linux
HOST_OS_EXTRA=Linux-5.8.0-44-generic-x86_64-Ubuntu-20.10
HOST_CROSS_OS=windows
HOST_CROSS_ARCH=x86
HOST_CROSS_2ND_ARCH=x86_64
HOST_BUILD_TYPE=release
BUILD_ID=QQ3A.200605.002.A1
OUT_DIR=out
PRODUCT_SOONG_NAMESPACES=external/mesa3d
============================================
root@ziomario-Z390-AORUS-PRO:/home/ziomario/Scrivania/Data/Nano/I9/aosp# mkdir {system,vendor}-r{o,w}
root@ziomario-Z390-AORUS-PRO:/home/ziomario/Scrivania/Data/Nano/I9/aosp# dd if=/dev/zero of=system-rw.img bs=512 count=6291456
6291456+0 records in
6291456+0 records out
3221225472 bytes (3.2 GB, 3.0 GiB) copied, 6.7983 s, 474 MB/s
root@ziomario-Z390-AORUS-PRO:/home/ziomario/Scrivania/Data/Nano/I9/aosp# mkfs.ext4 system-rw.img -L "/"
mke2fs 1.45.6 (20-Mar-2020)
Discarding device blocks: done                            
Creating filesystem with 786432 4k blocks and 196608 inodes
Filesystem UUID: b4f3bcb5-1837-4c57-9134-1f3e9161b08c
Superblock backups stored on blocks: 
        32768, 98304, 163840, 229376, 294912

Allocating group tables: done                            
Writing inode tables: done                            
Creating journal (16384 blocks): done
Writing superblocks and filesystem accounting information: done 

root@ziomario-Z390-AORUS-PRO:/home/ziomario/Scrivania/Data/Nano/I9/aosp# sudo mount system-rw.img system-rw
root@ziomario-Z390-AORUS-PRO:/home/ziomario/Scrivania/Data/Nano/I9/aosp# sudo mount out/target/product/generic_arm64/system.img system-ro -o ro
mount: /home/ziomario/Scrivania/Data/Nano/I9/aosp/system-ro: special device out/target/product/generic_arm64/system.img does not exist.

**Comment by Marietto2008** [comment]
@pgielda

**Comment by pgielda** [comment]
do you actually issue make ("m") at any point? It should be the next command after "lunch aosp_arm64-eng"...

**Comment by Marietto2008** [comment]
yes,I've issued make -j4 when I read (here : https://source.android.com/setup/build/gsi#building-gsis) that it had to be done. you know,not everybody knows what command is "m". I suggest to you to change the "m" with a nice make.

**Comment by kgugala** [comment]
@Marietto2008 `m` is the tool you should use. And if you're working with AOSP you should know it (see: https://source.android.com/setup/build/building#build-the-code)

**Comment by Marietto2008** [comment]
I used make -j4 and it has compiled succesfully.
